### PR TITLE
bugfixes for json dummy module

### PR DESCRIPTION
**Pull request by arlimus**

### CODE REVIEW

File: lib/utils/parser.rb [hunk 1]
```diff
@@ -46,4 +32,20 @@ def parse_comment_line(raw, opts)
     end
     [line, idx_nl]
   end
+
+  private
+
+  # parse a etc/passwd line
+  def parse_passwd_line(line)
```
Comment (chris-rock) [inline lib/utils/parser.rb]: please make this public again. it is directly used in user resource https://github.com/chef/inspec/blob/master/lib/resources/user.rb#L222

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by chris-rock** [comment]
Great additions @arlimus
@delivery approve

**Comment by chef-delivery** [comment]
Change: 6ce18d67-c6d7-4488-89a3-3b48ed93be37 approved by: @chris-rock

**Comment by chef-delivery** [comment]
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

**Pull request merged**